packages/fxa-settings/src/components/Page2faReplaceRecoveryCodes/index.test.tsx

```diff
@@ -36,6 +36,8 @@ it('renders', async () => {
   expect(screen.getByTestId('2fa-recovery-codes')).toHaveTextContent(
     recoveryCodes[0]
   );
+
+  expect(screen.getByTestId('datablock').textContent?.trim()).toEqual(recoveryCodes.join(" "))
```

Comment:
This is more of a personal preference, but it's also shorter: instead of using a testid you could look up each code by its text using `multiValue.forEach((v) => screen.getByText(v))` - this more closely represents what a user sees.

Not a blocker, though!

Reply:
I guess the reason I did it this way is that I was explicitly checking that the 'textContent' had received spaces as per the separator prop. This is the crux of the issue.

Reply:
🤦 of course - my bad